types/context.go

```diff
@@ -80,6 +81,7 @@ func (c Context) VoteInfos() []abci.VoteInfo                    { return c.voteI
 func (c Context) GasMeter() storetypes.GasMeter                 { return c.gasMeter }
 func (c Context) BlockGasMeter() storetypes.GasMeter            { return c.blockGasMeter }
 func (c Context) IsCheckTx() bool                               { return c.checkTx }
+func (c Context) InConsensus() bool                             { return !c.outOfConsensus }
```

Comment:
Can we keep the naming consistent? Just call the field `inConsensus`.

Reply:
@alexanderbez I'm a bit confused,Should I just rename outOfConsensus to inConsensus or anything else?

Reply:
```go
func (c Context) InConsensus() bool                             { return c.inConsensus }
```